pkg/nsx/services/inventory/build.go

```diff
@@ -200,3 +200,58 @@ func (s *InventoryService) BuildNamespace(namespace *corev1.Namespace) (retry bo
 	}
 	return
 }
+
+func (s *InventoryService) BuildService(service *corev1.Service) (retry bool) {
```

Comment:
Do we need to update containerApplicationIds in Pods for ContainerApplication?
If not, maybe we would also like to remove the containerApplicationIds parts in BuildPod?

Reply:
@TaoZou1 +1